web/client/components/mapcontrols/annotations/CoordinatesEditor.jsx

```diff
@@ -195,7 +206,8 @@ class CoordinatesEditor extends React.Component {
                     if (this.props.type === "Polygon") {
                         tempComps = this.addCoordPolygon(tempComps);
                     }
-                    this.props.onChange(tempComps, this.props.properties.radius, this.props.properties.valueText);
+                    // todo verify this callback
```

Comment:
this comment is too generic. Can you describe the reason to verify this callback (or remove this comment)